br-webui/views/system.liquid

```diff
@@ -265,6 +277,11 @@ socket.on('restore px params complete', function(data){
 					<button type="button" class="btn btn-primary"
 						id="restore params" onclick="restoreParams()">
 						Restore Default Parameters</button>
+
+					<h4>Reboot Pixhawk</h4>
+					<button type="button" class="btn btn-primary"
+						id="reboot pixhawk" onclick="rebootPixhawk()">
+						🔃</button>
```

Comment:
What is this 'recycle' symbol?

Reply:
Recycle symbol: ♻

Reply:
I can modify to use only "Reboot" if you prefer.

Reply:
Ok I didn't realize that that symbol is the button's innerHTML, that's cool.